docs/content/strict.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Compiler Strict Mode
+kind: misc
+weight: 9
+---
+
+The Rego compiler supports `strict mode`, where an additional set of rules are enforced during compilation.
```

Comment:
Is `rule` the correct term here?

Reply:
I'd propose "where additional constraints and safety checks"... 🤔